### PR TITLE
[DEV-801] - Add GitBook quote block parsing

### PR DESCRIPTION
**Pull request by kin0992**
#### List of Changes

- Add new `blockquote` schema
- Change `RenderingComponents`

#### Motivation and Context

Add the parsing for kind of gitbook quote blocks

#### How Has This Been Tested?

Unit Test

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/gitbook-docs/src/__tests__/parseContent.test.ts [hunk 1]
```diff
@@ -164,4 +164,16 @@ describe('parseContent', () => {
...
     ]);
   });
+
+  xit('should parse quote', () => {
+    const quote =
+      '> _« ... al fine di consentire le attività di riconciliazione del pagamento... »_';
+    expect(parseContent(quote, config)).toStrictEqual([
+      new Markdoc.Tag('Quote', {}, [
+        new Markdoc.Tag('Paragraph', {}, [
+          '« ... al fine di consentire le attività di riconciliazione del pagamento... »',
```
Comment (datalek) [inline packages/gitbook-docs/src/__tests__/parseContent.test.ts:174]: I see that the `_` char is missing, is the correct behaviour?

File: packages/gitbook-docs/src/__tests__/parseContent.test.ts [hunk 2]
```diff
@@ -164,4 +164,16 @@ describe('parseContent', () => {
       ),
     ]);
   });
+
+  xit('should parse quote', () => {
```
[the remark this answers is not in the document]
Comment (kin0992) [reply packages/gitbook-docs/src/__tests__/parseContent.test.ts:168]: Yep, I used `xit` just to skip that test and not having the CI failing. I'm gonna remove it when I've completed the activity.

File: packages/gitbook-docs/src/__tests__/parseContent.test.ts (hunk 1 above)
Comment (kin0992) [reply packages/gitbook-docs/src/__tests__/parseContent.test.ts:174]: Well, the parse wraps the text (without the `_`) in an `em` node, but I think we should both handle `> _« ... al fine di consentire le attività di riconciliazione del pagamento... »_` and `> « ... al fine di consentire le attività di riconciliazione del pagamento... »`

**Review by datalek — APPROVED** [review]
🥇

**Pull request merged**